plugin/indent_blankline.vim

```diff
@@ -29,8 +28,6 @@ command! IndentBlanklineToggleAll call indent_blankline#commands#ToggleAll()
 
 augroup IndentBlanklineAutogroup
     autocmd!
-    autocmd Syntax * IndentBlanklineRefresh
-    autocmd BufWritePost * IndentBlanklineRefresh
-    autocmd FileChangedShellPost * IndentBlanklineRefresh
     autocmd OptionSet shiftwidth,tabstop IndentBlanklineRefresh
+    autocmd FileChangedShellPost,Syntax,TextChanged,TextChangedI,WinScrolled * IndentBlanklineRefresh
```

Comment:
You need a guard for `WinScrolled` unless your requirements are *nighlty build only*.

Reply:
I am waiting to merge until 0.5 is released